about.html

```diff
@@ -0,0 +1,216 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;1,900&display=swap"
+      rel="stylesheet"
+    />
+
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <link
```

Comment:
- [ ] **Optional:** Kindly fix the console errors shown below.

![image](https://user-images.githubusercontent.com/27709832/153589921-8fca9ca7-80b3-46f3-b699-44a9337dd9c0.png)

Reply:
thanks for noticing the console errors 😄 , I have fixed it.